Tests/UnitTests/Sources/GREYActionsTest.m

```diff
@@ -198,5 +198,35 @@ - (void)testInvalidTapActionSucceedsAfterDisablingConstraints {
   }
 }
 
+- (void)testReplaceTextSucceedsOnUITextField {
+    NSString *textToReplace = @"A String";
+    NSError *error;
+
+    UITextField *textfield = [[UITextField alloc] init];
+
+    id<GREYAction> replace = [GREYActions actionForReplaceText:textToReplace];
+    [replace perform:textfield error:&error];
+
+    XCTAssertNil(error);
+
+    XCTAssertEqualObjects(textToReplace, textfield.text);
+}
+
+- (void)testReplaceTextFailsOnUIView {
+    NSString *textToReplace = @"A String";
+    NSError *error;
+
+    UIView *label = [[UIView alloc] init];
+
+    id<GREYAction> replace = [GREYActions actionForReplaceText:textToReplace];
+    [replace perform:label error:&error];
+
+    NSString *reasonSubstring = @"Action could not be performed on element";
```

Comment:
I can foresee this coming up in a bit. We probably want to create a different error reason here.